markdown-format/README.md

```diff
@@ -23,22 +23,51 @@ Formatting markdown with one sentence per line makes it easier to:
 
 ## Installation
 
+### With Go
+
 ```bash
 go build
 ```
 
+### With mise
+
+Install using [mise](https://mise.jdx.dev/) with the Go backend:
+
+```bash
+mise use -g go:github.com/neongreen/mono/markdown-format@main
+```
+
+Or add to your `.mise.toml`:
+
+```toml
+[tools]
+"go:github.com/neongreen/mono/markdown-format" = "main"
+```
+
 ## Usage
 
-Format a markdown file:
+Format a markdown file to stdout:
```

Comment:
i want formatting inplace to be the default, and formatting stdin->stdout to be if called without arguments.

and should also support `--check` like rustfmt does

and also also the wrapper script shouldnt be needed for dprint either, it just generally shouldnt be used